processes/wps_say_hello.py

```diff
@@ -0,0 +1,64 @@
+from pywps import Process, LiteralInput, LiteralOutput, UOM
+from pywps.app.Common import Metadata
+
+import logging
+
+LOGGER = logging.getLogger("PYWPS")
+
+
+class SayHello(Process):
```

Comment:
I think we could just call this `TestProcess`.  Furthermore we can simplify things a bit since we aren't actually using this as a real process.  See comments below.